dart/lib/src/enricher/io_enricher_event_processor.dart

```diff
@@ -55,7 +54,6 @@ class IoEnricherEventProcessor extends EventProcessor {
     // Pure Dart doesn't have specific runtimes per build mode
     // like Flutter: https://flutter.dev/docs/testing/build-modes
     final dartRuntime = SentryRuntime(
-      key: 'sentry_dart_runtime',
```

Comment:
Is that right? the runtime needs a `key`.

Reply:
No, a runtime doesn't need a key. It's just required if there are multiple runtimes with the same name, but that should never be the case.

Reply:
Yeah indeed, just wondering why this is removed.

Reply:
It looks better in the UI without the key :D

Reply:
What's about `sentry_flutter_runtime` then?

Reply:
Oh also removed, ok, closing it.